docs/dev-concepts/faq.md

```diff
@@ -87,21 +88,25 @@ There are no message storage and retrieval-related fees incurred by developers f
 
 ### How does XMTP establish a secure and fraud-proof relationship between two identities?
 
-Blockchain accounts sign and advertise a set of keys that XMTP uses to establish a shared secret for encryption using another blockchain account’s keys. These keys attest to the authenticity of both accounts and are required to add messages to their conversation. No third-party apps or relayers are involved in this process.
+Blockchain accounts sign and advertise a set of keys to start using XMTP. These keys attest to the authenticity of both accounts. XMTP uses these keys to establish a shared secret between the blockchain accounts and uses it to generate a key used to encrypt an invitation that allows the blockchain accounts to start exchanging messages. No third-party apps or relayers are involved in this process.
 
-To learn more about signatures, see [Sign to send and receive messages using apps built with XMTP](account-signatures).
+To learn more about these keys, see [Key generation and usage in XMTP](key-generation-and-usage).
+
+To learn more about invitation and message encryption, see [Invitation and message encryption with XMTP](invitation-and-message-encryption).
 
 ### Does each blockchain account address have a corresponding XMTP identity?
 
-Yes. Each blockchain account address is represented by an XMTP identity key. This identity key is a part of a key bundle that only that the address can use to authenticate messages.
+Yes. Each blockchain account address is represented by an XMTP identity key. This identity key is a part of a key bundle that only the address that generated it can use to authenticate messages.
 
-To learn more about XMTP identities, see [Sign to send and receive messages using apps built with XMTP](account-signatures).
+To learn more about XMTP identity keys, see [Key generation and usage in XMTP](key-generation-and-usage).
 
 ### Do apps built with XMTP need to decrypt messages with blockchain account private keys each time?
 
-When a user starts a new messaging session with an app built with XMTP, the user must sign with their blockchain account private key to decrypt their XMTP key bundle, which is then used for message decryption. A one-time signature is also required to create that key bundle, which includes an XMTP identity.
+When a user starts a new messaging session with an app built with XMTP, the user must sign with their blockchain account private key to decrypt their XMTP key bundle, which is then used for invitation and message decryption. A one-time signature is also required to create that key bundle, which includes an XMTP identity.
```

Comment:
@mkobetic I made these refinements to reflect XMTP V2 - any thoughts?

Reply:
The change is OK, but my answer to the question would be "No, the account key is only used for signing". The phrasing of the question may be a bit off, what did we want to convey with this one?

Reply:
Adjusted text per chat with Martin.